README.md

```diff
@@ -40,6 +40,7 @@ Initial implementations include:
  - [Java implementation](https://github.com/apache/arrow/tree/master/java)
  - [C++ implementation](https://github.com/apache/arrow/tree/master/cpp)
  - [Python interface to C++ libraries](https://github.com/apache/arrow/tree/master/python)
+ - [JavaScript reader and writer](https://github.com/apache/arrow/tree/master/js)
```

Comment:
Could we just say "JavaScript implementation" here to be consistent?

Reply:
+1